docs/examples/viz_slicer.py

```diff
@@ -0,0 +1,40 @@
+import numpy as np
+from fury import actor, window
+from fury.utils import get_slices, show_slices
+from dipy.data import read_mni_template
+
+###############################################################################
+# Let's read the 3D data from dipy
+
+nifti = read_mni_template()
+data = np.asarray(nifti.dataobj)
```

Comment:
not sure an user in mechanical fluid understand this. You have to explain more or use a more general data.

Why `nifti.dataobj`? etc